Source/JavaScriptCore/assembler/MacroAssemblerX86Common.cpp

```diff
@@ -740,7 +739,9 @@ asm (
     "popq %rcx" "\n"
     "popq %rbp" "\n"
     "ret" "\n"
+#if !COMPILER(MSVC)
```

Comment:
I should ask though: Is there any reason to bother inserting these? Is it just because we haven't removed `COMPILER(MSVC)`?

Reply:
clang-cl does not support this directive.